### ISSUE TITLE
verify  2.0.0 -> 2.0.1 migration from a dumped 2.0.0 database works

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#586]
this will be near last.  Worst-case we use the database-migration plugin.

https://grails.org/plugin/database-migration

It is basically setup for something like this at the end (as needed):

```
databaseChangeLog(logicalFilePath: "appName-autobase") {
   include "./migrations/batch/SomeBigChanges.groovy"
   include "./migrations/changelog-1.0.1.groovy"
   include "./migrations/changelog-1.0.5.groovy"
}
```

**Comment by nathandunn** [comment]
@deepakunni3   @cmdcolin   

As an FYI the 2.0.1-snapshot on the staging server from a dump from 2.0.0 and did not notice any issues.

**Comment by nathandunn** [comment]
Added blocked as its the last thing we do.

**Comment by nathandunn** [comment]
- [ ] test 2.0.0 -> 2.0.1 migration from dumped database

**Comment by nathandunn** [comment]
@cmdcolin  @deepakunni3  If you have anything extra just add it to the changelog  . . I went through the final pass with your changes and I think all of the improvements / fixes in 2.0.1 are accounted for.  Thanks.
